src/read/pe/section.rs

```diff
@@ -340,6 +346,19 @@ impl pe::ImageSectionHeader {
             .read_error("Invalid PE section offset or size")
     }
 
+    /// Tests whether a given RVA is part of this section
+    pub fn contains_rva(&self, va: u32) -> bool {
+        let section_va = self.virtual_address.get(LE);
+        match va.checked_sub(section_va) {
+            None => false,
+            Some(offset) => {
+                let (_section_offset, section_size) = self.pe_file_range();
```

Comment:
Are you sure you want the file size rather than the virtual size? I think the virtual size is more likely to be what the user expects.

Reply:
Indeed, thanks.